src/components/tBTC/BridgeActivity.tsx

```diff
@@ -59,13 +61,13 @@ export const BridgeActivity: FC<BridgeActivityProps> = ({
   emptyState,
   children,
 }) => {
-  const isBridgeHistoryEmpty = data.length === 0
+  const { active } = useWeb3React()
 
   return (
     <BridgeActivityContext.Provider
       value={{
         data,
-        isBridgeHistoryEmpty,
+        isWalletConnected: active,
```

Comment:
Non-blocking:

`isWalletConnected` is not really related to `BridgeActivity` so I'm wondering if it's worth placing it in the context? Maybe we could just call `const { active } = useWeb3React()` where it's needed?

Reply:
Ref commit: cfb65f4d